### DISCUSSION TITLE
Open link to mhtml file in Markdown

### DISCUSSION THREAD

**Discussion by Marinorama** [gsantner/markor#1866] [Q&A] [answered]
A link in Markdown to a local mhtml file does not open the file in standard browser (Opera in this case) although it should do so. At least it does in other applications. In Markor the link opens the share mode and - after chosen Opera - it does not open the mhtml file there but asks for the location to download it... How can I change the behaviour? I want to click the link in Markor/md and it should open the mhtml file in the standard browser.

**Comment by gsantner** [comment]
Chrome custom tabs settings option is maybe what you are after?

**Reply by Marinorama** [reply]
I checked the option Chrome custom tabs - but still link opens the "Share to..." menu instead of opening browser.

**Reply by gsantner** [reply, marked as answer]
is that what you link to a real file on your system, or a file on the web (as in a http(s):// link? Well in case it's a local file it behaves just like for all file formats Markor does NOT know about. It asks you to open it with other app and allowing you to chose one.

Markor doesn't have anything special implemented for what you ask about. It just asks in the same way you would refere a pdf.

**Reply by Marinorama** [reply]
Ok. I am just a little bit confused because choosing Opera in the share mode that Markor shows does not open/show the mhtml file (although Opera can and should) but lets Opera ask me where to download it.
